src/index.html

```diff
@@ -20,8 +20,380 @@
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="card">
+      <section
+        class="card__section"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        />
+        <h1 class="card__header">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--5">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__info">
+          <p class="card__content">Price:</p>
+          <p class="card__sale">$2,199</p>
+        </div>
+
+        <button
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          Buy
+        </button>
+      </section>
+
+      <section class="card__section">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27"
+          class="card__image"
+        />
+        <h1 class="card__header">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h1>
+        <p class="card__code">Product code: 195434</p>
+
+        <div class="card__raiting">
+          <div class="stars stars--5">
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+            <span class="stars__star"></span>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+
+        <div class="card__info">
+          <p class="card__content">Price:</p>
+          <p class="card__sale">$2,199</p>
+        </div>
+
+        <a
+          href="#"
+          class="card__link"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
```

Comment:
Duplicated code: You are repeating the same structure for each product card. This can be improved by using a loop to generate the cards dynamically. This will make your code more maintainable and easier to read.